curriculum/challenges/english/25-front-end-development/workshop-email-simulator/6853f4ded9f84896ff182dd8.md

```diff
@@ -0,0 +1,120 @@
+---
+id: 6853f4ded9f84896ff182dd8
+title: Step 19
+challengeType: 20
+dashedName: step-19
+---
+
+# --description--
+
+After the empty inbox check, add a print statement with the message `'\nYour Emails:'`. After that, iterate over all emails in the inbox using a `for` loop. Use enumeration to number the emails starting from `1`. Unpack the enumeration into `i` (the index) and `email` (the email object). Inside the loop, print each email using the `__str__` method you defined earlier, formatted with its number like this:
+
+```md
+1. [Formatted email summary from __str__ method]
+2. [Formatted email summary from __str__ method]
+.
+.
+.
+n. [Formatted email summary from __str__ method]
```

Comment:
I think a one-line example is fine here. Still I don't like to have all that text in the example.
```suggestion
[index]. [string representation of email object]
```